### PR TITLE
remove validation that devices_ordered must be <= cap

### PR DESCRIPTION
**Pull request by aldavidson**
### Context

See [Trello card 716](https://trello.com/c/FxejJUlY/716-remove-the-validation-that-devicesordered-cannot-be-greater-than-cap)

### Changes proposed in this pull request

* remove the validation which enforces cap >= devices_ordered

### CODE REVIEW

File: app/form_objects/support/enable_orders_form.rb [hunk 1]
```diff
@@ -7,7 +7,6 @@ class Support::EnableOrdersForm
 
   validates :order_state, inclusion: { in: School.order_states }
   validates :cap, numericality: { only_integer: true, greater_than: 0 }, if: :cap_required?
-  validates_with OrderStateAndCapValidator
```
Comment (benilovj) [inline app/form_objects/support/enable_orders_form.rb:10]: Do we not need to do `validate :cap_lte_allocation` here as well?

**Review by benilovj — APPROVED** [review]
Fine, bar the comment.

File: app/form_objects/support/enable_orders_form.rb (hunk 1 above)
Comment (aldavidson) [reply app/form_objects/support/enable_orders_form.rb:10]: Yeah, I guess we should - done

**Review by benilovj — APPROVED** [review]

**Comment by benilovj** [comment]
Looks good, please squash the commits if possible.

**Pull request merged**